rrdd/xcp_rrdd.ml

```diff
@@ -475,16 +475,28 @@ let uuid_blacklist = [
 module IntSet = Set.Make(Int)
 
 let domain_snapshot xc =
-  let dom_uuid_domid_of_domain dom =
+  let metadata_of_domain dom =
     let uuid = Uuid.(to_string (uuid_of_int_array (dom.Xenctrl.handle))) in
     let domid = dom.Xenctrl.domid in
     let start = String.sub uuid 0 18 in
 
+    (* Actively hide migrating VM uuids, these are temporary and xenops
+       writes the original and the final uuid to xenstore *)
+    let stable_uuid path = function
+      | Some key ->
```

Comment:
Is this correct? The parameter `path` is not used and shadowed by a local definition.

Reply:
I can rewrite it to avoid shadowing of names, I think it still was correct.

Reply:
But if the parameter is not used then it can be dropped.

Reply:
sorry, you're right, it was wrong.